### ISSUE TITLE
Timeline: add event not called

### ISSUE THREAD

**Issue by RalfScania** [primefaces/primefaces#7449]
Primefaces: 10.0.2

Description of the bug
Timeline add event is not called

Steps to reproduce
Reproducer: https://github.com/RalfScania/primefaces-test-pf10

If you press on a free space in the timeline, following event is never called in. TestView.
```
public void onAdd(TimelineAddEvent e) {
        FacesContext facesContext = FacesContext.getCurrentInstance();
        facesContext.addMessage(null, new FacesMessage(FacesMessage.SEVERITY_INFO, "add", "Test"));
    }
```

This was working in 10.0.0

**Comment by melloware** [comment]
Thanks for the reproducer!

**Comment by melloware** [comment]
I think it might be this issue: https://github.com/visjs/vis-timeline/pull/1081 in Timeline 7.4.8 that was fixed in 7.4.9.  Let me try it.

**Comment by melloware** [comment]
Confirmed its fixed in Timeline 7.4.9

**Comment by melloware** [comment]
@RalfScania looks like this was just released in 10.0.3 Elite if you want to give it a shot.

**Comment by RalfScania** [comment]
Works as expected. Thanks!
